db/migrate/20200420143824_move_electronic_pharmacy_stock_rows_into_files.rb

```diff
@@ -0,0 +1,18 @@
+class MoveElectronicPharmacyStockRowsIntoFiles < ActiveRecord::Migration[5.2]
+  class ElectronicPharmacyStockRecord < ApplicationRecord
+    has_one_attached :rows_file
+  end
+
+  def up
+    ElectronicPharmacyStockRecord.all.each do |e|
```

Comment:
Calling `all` will load all the records at once into memory. To avoid this use `find_each` (https://www.fastruby.io/blog/rails/performance/writing-fast-rails-part-3.html)
```suggestion
    ElectronicPharmacyStockRecord.find_each do |e|
```